content/Kotlin/kotlin.md

```diff
@@ -0,0 +1,18 @@
+---
+Title: 'Kotlin'
+Description: 'Kotlin is a contemporary language released by JetBrains in 2011. Kotlin is becoming popular and is fast replacing Java as people’s choice for software projects.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+[Kotlin](https://www.codecademy.com/catalog/language/kotlin) is a contemporary language released by JetBrains in 2011. Kotlin is becoming popular and is fast replacing Java as people’s choice for software projects. Java and Kotlin are compatible and proficiency in Java makes learning Kotlin much easier. 
+
+##Uses of Kotlin
+
+Back-end web development
```

Comment:
This should be a bulleted list with an introductory sentence.

Reply:
@SSwiniarski noted. Thanks